plugins/inventory/foreman.py

```diff
@@ -0,0 +1,279 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016 Guido Günther <agx@sigxcpu.org>, Daniel Lobato Garcia <dlobatog@redhat.com>
+# Copyright (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: foreman
+    plugin_type: inventory
+    short_description: foreman inventory source
+    version_added: "2.6"
```

Comment:
As it is about to be ejected from ansible, this information has no value anymore.
```suggestion
```

Reply:
I don't know what the current policy is with collections. It could become the collection version, which would still have value to the user.

If it is the collection version, then I can say for sure that 0.5.0 does not have any of this.

Reply:
AFAIK the sanity tests explicitly don't test/expect version_added inside collections.

I don't strictly care if this remains here for now or not, regardless of the value.